src/main/java/com/sap/sgs/phosphor/fosstars/model/rating/oss/OssSecurityRating.java

```diff
@@ -81,30 +86,39 @@ protected SecurityLabel label(double score) {
     /**
      * The default thresholds.
      */
-    public static final Thresholds DEFAULT = new Thresholds(5.0, 8.0);
+    public static final Thresholds DEFAULT = new Thresholds(5.0, 8.0, 8.0);
```

Comment:
Is it possible that unclear threshold may be too high? Is Confidence usually `>8.0`?

Reply:
I even think it could be higher. The rating is currently based on 26 features. If the threshold is 0.8, than it is okay if approximately up to 5 features are unknown (well, it actually depends on weights).

Confidence for most of projects in the report is more than 9:

https://github.com/SAP/fosstars-rating-core/blob/master/src/main/jupyter/oss/security/SecurityRatingAnalysis.ipynb

Reply:
ok 👍